spec/models/match_spec.rb

```diff
@@ -1,75 +1,71 @@
 require "rails_helper"
 
 RSpec.describe Match do
-  let(:match) do
-    build(:match, points_for_team1: 8,
-                  points_for_team2: 2)
-  end
+  let(:match) { build(:match) }
   let(:user) { create(:user) }
-  context "is valid" do
-    it "has team_1 points" do
-      expect(match.points_for_team1).to eq(8)
-    end
-
-    it "has team_2 points" do
-      expect(match.points_for_team2).to eq(2)
-    end
-
-    it "does have team_1" do
-      expect(match.team_1).to be_present
+  context "When it's valid" do
+    it "it's attributes are present and correct" do
+      expect(match).to be_valid
```

Comment:
no need for this it description since the context will say this so you only need the `it '' do ... end`